### DISCUSSION TITLE
Turn off the dialog in the presentation mode in manim-slides

### DISCUSSION THREAD

**Discussion by torannn** [jeertmans/manim-slides#351] [Q&A] [answered]
When I enter the presentation mode with `manim-slides present` in manim-slides, there's a dialog looks like this. How can I turn it off.
![image](https://github.com/jeertmans/manim-slides/assets/86940649/b243dff0-456e-4be4-a4ee-088d6b89110a)

**Comment by jeertmans** [comment, marked as answer]
Hello! This is already part of Manim Slides, but currently not released.

You can install [v5.1.0-rc1](https://github.com/jeertmans/manim-slides/releases/tag/v5.1.0-rc1), or even [v5.1.0-rc2](https://github.com/jeertmans/manim-slides/releases/tag/v5.1.0-rc2), and use `--hide-info-window` for that purpose!

What you can also use, is to use <kbd>ALT</kbd>+<kbd>TAB</kbd> to switch the order of windows such that the presentation window appears on top of the info window.
